Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION = 1.1.1
+VERSION = 1.1.2
```

Comment:
Keep the version as is, update the version in another PR specifically for cutting the new release after internal testing.

Reply:
👍  A merge will create a new image under the git commit short hash. We can use it for internal soaking before releasing a new version.

Reply:
Reverted back to 1.1.1